client/app/components/form/UserOperatorMultiStepForm.tsx

```diff
@@ -101,12 +100,27 @@ export default function UserOperatorMultiStepForm({
   const isFormStatusDisabled =
     formData?.status === Status.PENDING || formData?.status === Status.APPROVED;
 
+  const isReviewWithRequestChanges =
+    isCasInternal && formData.is_new && formSection === 1;
+  const isReviewWithoutRequestChanges = isCasInternal && formSection === 2;
+  const isNewOperatorMessage =
+    !isCasInternal && formData.is_new && formSection === 1;
+
   const operatorRoute = isCasInternal ? "operators" : "select-operator";
   // If the user is an approved cas internal user or if no operator exists show the entire multistep form
   if (isCasInternal || !userOperatorId || formSection) {
     return (
       <>
-        {isCasInternal && formData.is_new && formSection === 0 && (
+        {isNewOperatorMessage && (
+          <Note
+            classNames={"mb-4 mt-6"}
+            showNotePrefix={false}
+            showAlertIcon={true}
+            message="Please fill out missing information or update incorrect information about your operator to continue with your application.
+Some fields cannot be edited. If you need to change those fields, please contact us via email at GHGRegulator@gov.bc.ca."
+          />
+        )}
+        {isReviewWithRequestChanges && (
```

Comment:
Moved this here to conditionally display is. To test this you can log in as `bc-cas-dev-third`, click `Add operator`, submit the whole form and then return to the page.

Reply:
Results of test "log in as bc-cas-dev-third, click Add operator, submit the whole form and then return to the page"

![image](https://github.com/bcgov/cas-registration/assets/120038448/a0c6ccec-6b09-4d46-bcaa-7343642878b1)

Reply:
> Results of test "log in as bc-cas-dev-third, click Add operator, submit the whole form and then return to the page"

Hey Shon, thanks for all the effort reviewing this to help us squash these bugs. I have just tested this twice and I do get the alert message. Can you please make sure you reset the client server if you switch branches? I just switched from another branch and noticed that the alert message was always showing until I reset the client. Must be a NextJS cache issue.

My results for both times:

<img width="1454" alt="Screenshot 2024-01-25 at 9 21 49 AM" src="https://github.com/bcgov/cas-registration/assets/14259474/b2786c35-65b2-49cd-a006-af1f762ef1ed">

Reply:
@marcellmueller 
I am unsure of the expected result of test:  "log in as bc-cas-dev-th(ree), click Add operator, submit the whole form and then return to the page".  I expected to see the "Please fill out missing info" message but, I do not see the message (as per screen shots above)

Reply:
I'm sorry I have no idea what it could be, I have tested it multiple times. To be clear this only works when you go through `Add operator` workflow and not `Select operator` since it only shows up for new operators. If you want to sync today let me know.